cluster/gce/configure-vm.sh

```diff
@@ -585,12 +587,21 @@ function create-salt-master-auth() {
     # NB: If this list ever changes, this script actually has to
     # change to detect the existence of this file, kill any deleted
     # old tokens and add any new tokens (to handle the upgrade case).
-    local -r service_accounts=("system:scheduler" "system:controller_manager" "system:logging" "system:monitoring")
     for account in "${service_accounts[@]}"; do
       token=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
       echo "${token},${account},${account}" >> "${KNOWN_TOKENS_FILE}"
     done
   fi
+  if [ ! -e "${ABAC_POLICY_FILE}" ]; then
+    local -r system_identites=("admin" "kubelet" "kube_proxy" "kubecfg" "client")
```

Comment:
`kubecfg` is the CN of the client cert we make for kubectl in kube-up.sh: https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/generate-cert/make-ca-cert.sh#L97

`client` is the CN for the cert we generate in GKE.